File: ScriptList.py
import Script
class ScriptList:
    def __init__(self, file):
        self.importScripts(file)
    def importScripts(self, file):
        try:
            scripts = []
            f=open(file, "r")
            f1 = f.readlines()
            for x in f1:
                script = Script.Script(x)
                scripts.append(script)
        except FileNotFoundError:
            print("File not found")

File: src/ScriptList.java
package botPackage;

import java.io.File;
import java.io.FileNotFoundException;
import java.util.ArrayList;
import java.util.Scanner;

// List that holds prewritten scripts we create

public class ScriptList {
	// array list to holds list of scripts
	ArrayList<Script> scripts = new ArrayList<Script>();
	
	public ScriptList(File file) {
		importScripts(file);
	}

	private void importScripts(File file) { 
		// For Example: file = QuestionScripts.txt 
		// Scanner grabs all the scripts from said text file
		try(Scanner in = new Scanner(file);) {
			// read list of scripts from a test file
			//in.useDelimiter("\n");//new script every line break
			String script = in.nextLine();
			scripts.add(new Script(script));
			
		} catch (FileNotFoundException e) {
			e.printStackTrace();
		}
		
	}
	
}
